nengo/neurons.py

```diff
@@ -545,9 +545,10 @@ def __init__(self, tau_n=1, inc_n=0.01, **lif_args):
 
     @property
     def _argreprs(self):
-        args = super(AdaptiveLIFRate, self)._argreprs
+        args = []
         self._add_argrepr(args, 'tau_n', 1)
         self._add_argrepr(args, 'inc_n', 0.01)
+        args.extend(super(AdaptiveLIFRate, self)._argreprs)
```

Comment:
I don't understand why switching this order makes it consistent. What was inconsistent about it before? Other neuron types (e.g. LIF) have the base class parameters first, then the unique parameters.

Reply:
Everything is now in the same order as in the constructor. Can you give an example of a unit test that you think should give different results?

Reply:
Ahh, I just didn't understand what you were using as your metric for "consistent". Having them in the same order as the constructor makes sense.